pages/en/lb4/Installation.md

```diff
@@ -12,111 +12,26 @@ summary:
 
 Install the following:
 
-- [Node.js](https://nodejs.org/en/download/) version 7 or higher.
+- Node.js, either:
+  - [Version 6.x](https://nodejs.org/en/download/): The long-term support (LTS) version.
+  - **Recommended**: [Version 8.x](https://nodejs.org/en/download/current/): The latest/current version.
 - [TypeScript](https://www.typescriptlang.org/index.html#download-links) version 2 or higher:
 
-   ```
-   npm i -g typescript
-   ```
-- [TypeScript Node](https://github.com/TypeStrong/ts-node#installation) version 3 or higher:
+  ```
+  $ npm i -g typescript
```

Comment:
As was pointed out earlier, global dependencies are an anti-pattern in Node.js world. Projects should list their build tooling dependencies in their package.json and then provide scripts like `npm run build` to invoke them. This is possible because when a package script is invoked by `npm`, `npm` automatically adds dev-dependencies to `PATH`.

I am ok to tell people to use globally installed `tsc` and `ts-node` to get them started quickly, but I think we should also tell them this is a shortcut not suitable for longer term, and show them how to setup their TypeScript project properly. The project https://github.com/strongloop/loopback4-extension-starter has a pretty good setup, I am not sure if other example projects have picked it up yet.

Reply:
OK, I added a note about this just being a shortcut and not best practice.